lib/Dancer2/Plugin/Swagger2.pm

```diff
@@ -224,8 +224,20 @@ sub _validate_request {
                 next;
             }
 
+
             my $value  = $values[0];
-            my %input  = ( $name => $value );
+
+            # TODO steal more from Mojolicious::Plugin::Swagger2 ;-)
+            if ($type and defined ($value //= $parameter_spec->{default})) {
```

Comment:
This looks error-prone: Might default values apply if `$type` is undefined? `$value` is modified in complex `if`-clause.

I'm gonna handle this.